docs/jupyter/bokeh.ipynb

```diff
@@ -0,0 +1,106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demonstrating Bokeh Modin Interoperability"
```

Comment:
Should we clarify that Modin is not interoperable with bokeh?

Reply:
I've added this to the subheading